packages/workflow/src/components/style.scss

```diff
@@ -0,0 +1,229 @@
+@use "@wordpress/base-styles/mixins" as *;
+@use "@wordpress/base-styles/variables" as *;
+@use "@wordpress/base-styles/colors" as *;
+@use "sass:color";
+
+// dirty hack to clean up modal
+.workflows-workflow-menu {
+	border-radius: $grid-unit-05;
+	width: calc(100% - #{$grid-unit-40});
+	margin: auto;
+	max-width: 400px;
+	position: relative;
+	top: calc(5% + #{$header-height});
+
+	@include break-small() {
+		top: calc(10% + #{$header-height});
+	}
+
+	.components-modal__content {
+		margin: 0;
+		padding: 0;
+	}
+}
+
+.workflows-workflow-menu__overlay {
+	display: block;
+	align-items: start;
+}
+
+.workflows-workflow-menu__header {
+	display: flex;
+	align-items: center;
+	padding: 0 $grid-unit-20;
+
+	.components-button {
```

Comment:
Which parts of these overrides are required, and which can be minimized?

I'd love to have @WordPress/gutenberg-components look at these just in case.

Reply:
Good point, this is also copied from the `@wordpress/commands` package and I don't see that rendering any buttons, granted that code is 3 years old. I will review the whole styles and see what I can trim from it.